src/Qowaiv.Validation.TestTools/README.md

```diff
@@ -39,4 +38,14 @@ as `Result` is defined to be invalid bases on containing at least one error
 message. Furthermore, it does not exposes the `.Value` as that is not
 accessible for an invalid result.
 
-All (except for the `.Value`) is also for the non-generic `Result`.
+All (except for the `.Value`) is also applicable for the non-generic `Result`.
+
+## Validate with
+There is also some syntactic sugar for testing validators on a model of choice:
+
+``` C#
+var model = new MyModel();
+
+Result<MyModel> result = model.ValidateWith(new ValidatorForMyModel);
```

Comment:
I suggest adding that you can chain this with a Should.

So "x,Should().BeInvalidFor<y>()" "becomes x.ValidateWith(new y).Should().BeInvalid()"

This will help with people running into this when updating and just needing a replacement for their code.